.conda-envs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: "pyatlan"
+  version: "{{ environ['GIT_DESCRIBE_TAG'] }}"
+
+source:
+  path: ../../
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pip
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
+  run:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
```

Comment:
Same doubt here ☝️(similar to: https://github.com/atlanhq/atlan-python/pull/371/files#r1715331215)